lighthouse-core/test/lib/csp-evaluator-test.js

```diff
@@ -16,63 +15,6 @@ const {
 
 /* eslint-env jest */
 
-describe('Evaluator compatibility', () => {
```

Comment:
This was just a sanity check since the previous method of importing CSP evaluator was irregular. We shouldn't need it now.